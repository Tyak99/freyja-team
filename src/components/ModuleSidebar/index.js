import ModuleSidebar from './ModuleSidebar';

export default ModuleSidebar;
