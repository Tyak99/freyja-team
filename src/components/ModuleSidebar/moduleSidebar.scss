$sidebar-links-color: #8f5806;
$dark-brown-color: #523304;

.m-aside {
  margin-top: 41px;
  background: #f8f8f8;
  border: solid #f8f8f8 1px;
}

.d-aside{
  padding-top: 2rem;
  min-height: 100vh;
}

 .d-sidebar aside {
  padding: 10px;
}

.d-links li.module-active {
  background: $dark-brown-color;
  &:hover {
    border: 0;
  }
}

.d-links li.module-active a {
  color: #fff;
}
.d-links li a {
  text-decoration: none;
  color: $dark-brown-color;
}

ul.d-links {
  list-style: none;
  margin: 0;
  padding: 0;
}

.d-links li, span.d-logo {
  cursor: pointer;
  margin: 4rem 1rem;
  text-align: center;
  padding: 5px;
}

.d-links li:hover {
  border: 2px solid $sidebar-links-color;
  border-radius: 2px;
}

a.m-card-link {
  color: #000;
}
