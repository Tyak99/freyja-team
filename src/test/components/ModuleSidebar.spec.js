import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ModuleSidebar from '../../components/ModuleSidebar';

describe('Component: ModuleSidebar', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<ModuleSidebar />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
