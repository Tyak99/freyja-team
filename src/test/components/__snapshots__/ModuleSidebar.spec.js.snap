// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: ModuleSidebar should render correctly 1`] = `
<aside
  className="m-aside"
>
  <div
    className="d-aside"
  >
    <div>
      <span
        className="d-logo"
      >
        <img
          alt="logo"
          src="test-file-stub"
        />
        Community
      </span>
    </div>
    <ul
      className="d-links"
    >
      <li
        className="module-active"
      >
        <Link
          to="/dashboard"
        >
          Modules
        </Link>
      </li>
      <li>
        <Link
          to="/profile"
        >
          Profile
        </Link>
      </li>
      <li>
        <Link
          to="/payment"
        >
          Payment
        </Link>
      </li>
      <li>
        <Link
          to="/aboutus"
        >
          Help
        </Link>
      </li>
    </ul>
  </div>
</aside>
`;
